services/debugger/cache.go

```diff
@@ -0,0 +1,100 @@
+package debugger
+
+import (
+	"github.com/rudderlabs/rudder-server/config"
+	"sync"
+	"time"
+)
+
+/*
+	loadCacheConfig sets the properties of the cache after reading it from the config file.
+This gives a feature of hot readability as well.
+ */
+func (cache *CacheT) loadCacheConfig() {
+	config.RegisterIntConfigVariable(3, &cache.size, true, 1, "LiveEvent.Cache.size")
+	config.RegisterDurationConfigVariable(time.Duration(720), &cache.keyTTL, true, time.Hour,
```

Comment:
I am not sure if it a good idea to register these variables multiple times, cc @chandumlg